packages/client/components/AnalyticsPage.tsx

```diff
@@ -175,6 +182,10 @@ const AnalyticsPage = () => {
     cacheEmail().catch()
   }, [isSegmentLoaded])
 
+  // page titles are changed in child components via useDocumentTitle, which fires after this
+  // we must guarantee that this runs after useDocumentTitle
+  // we can't move this into useDocumentTitle since the pathname may change without chaging the title
+  const TIME_TO_RENDER_TREE = 100
```

Comment:
+1 I don't know if you've tested this, but for me with Safari, I need a bigger timeout to make this work correctly, at least on the first 2 renders:
* load `localhost:3000` -> it redirects to /meetings and tracks /meetings with signup title
* go to a team dashboard -> it tracks the dash path with "Meetings" title

After that it works fine. I assume it's because we're lazy loading some components, so it takes more time in the beginning.

Reply:
In practice, we almost always use path instead of page title when analyzing things.  I'm going to merge this as is and we can revisit the threshold if it becomes an actual problem.